hphp/util/word-mem.h

```diff
@@ -110,6 +110,19 @@ inline void bcopy32_inline(void* dst, const void* src, uint32_t len) {
                        : "+r"(len), "+r"(src), "+r"(dst)
                        :: "xmm0", "xmm1"
                       );
+#elif defined(__aarch64__)
+  int64_t t3, t4, t5, t6;
+  __asm__ __volatile__("lsr    %x0, %x0, #5\n"
+                       ".LBCP32%=:\n"
+                       "ldp    %x3, %x4, [%x1], #16\n"
+                       "ldp    %x5, %x6, [%x1], #16\n"
+                       "subs   %x0, %x0, #1\n"
+                       "stp    %x3, %x4, [%x2], #16\n"
+                       "stp    %x5, %x6, [%x2], #16\n"
+                       "bgt    .LBCP32%=\n"
```

Comment:
Likewise here. 
Also it would better if you only do the update of the addresses once rather for each stp.  On Cavium's newer cores, the stp with an update is split into two micro-op.